bandit/plugins/general_bind_all_interfaces.py

```diff
@@ -37,8 +37,7 @@
 
 .. seealso::
 
- - __TODO__ : add best practice info on binding to all interfaces, and link
-   here.
+ - https://github.com/Pylons/pyramid/issues/489
```

Comment:
I found another example which might be a little better.  

https://nvd.nist.gov/vuln/detail/CVE-2018-1281

It's again specific example of a project/product, but at least it was reported as a CVE.

Reply:
Yes that looks better, I'll make the change.